<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="MapIn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>导入地图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DataOut">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>导出数据</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="MapWidget" name="widget_map" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>631</width>
         <height>610</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>643</x>
         <y>9</y>
         <width>311</width>
         <height>610</height>
        </rect>
       </property>
       <widget class="QSplitter" name="splitter_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>79</y>
          <width>133</width>
          <height>161</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLineEdit" name="ID"/>
        <widget class="QLineEdit" name="Map"/>
        <widget class="QLineEdit" name="Major"/>
        <widget class="QLineEdit" name="PointID"/>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>101</width>
          <height>161</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="Label1">
         <property name="text">
          <string>ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="Label2">
         <property name="text">
          <string>Map</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="Label3">
         <property name="text">
          <string>Major</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>PointID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="load">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>载入</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>20</width>
          <height>381</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>401</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>260</y>
          <width>81</width>
          <height>61</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X轴：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="X">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>260</y>
          <width>71</width>
          <height>61</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Y轴：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="Y">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
